uwblueprint/website-bp
# Admin Dashboard - Top Header w/ Tabs and Admin Dashboard - Table - Role Header

## [Design Video](https://www.loom.com/share/647d8564027b43b9b81c6ae49ee89274) 
Covers:
* Logo button redirect
* Selecting different views in the dropdown
* Settings / profile buttons don't have functionality yet (we should cover this in later tickets)
* Department buttons - hover and active states
* Role buttons - hover and active states
* [A11y] Focus states when navigating by keyboard




### Notion ticket link

[Admin Dashboard - Top Header w/ Tabs](https://www.notion.so/uwblueprintexecs/Admin-Dashboard-Top-Header-w-Tabs-4b2d876710a5493a9baf99272d068787)

[Admin Dashboard - Table - Role Header](https://www.notion.so/uwblueprintexecs/Admin-Dashboard-Table-Role-Header-6a785ee555274c89817ca47ec204cd11)

### Implementation description

- Added header with tabs to the admin dashboard
- Added role header to the admin dashboard



check against https://www.figma.com/file/z0heicLtAk7PBC48xiD9hU/Application-Dashboard?node-id=195%3A1176&t=XLsBe30CVtHB2qLM-1

### Steps to test

1. Run yarn dev
2. Navigate to localhost:3000/admin



i didn't make the Users a prop just for testing but it should be pretty easy to do that.

### Checklist
